Remove background averaging

talonchandler opened the pull request:
When I was testing calibration using an ROI, I noticed that the background correction changed behavior. 

I found that when a calibration ROI was used, the background correction averaged over this ROI then applied that background correction uniformly over the whole image. I cannot think of a reason to average over a FOV within a background image, so I removed this functionality. 

This functionality severely reduces the effectiveness of the "Measured" background corrections when calibrating within an ROI. It's also likely that this bug was partially responsible for Yuming's poor background corrections.

ziw-liu commented:
Do you think a warning is still needed since we are altering the behavior of the pipeline?

talonchandler commented:
I *think* I removed the warning and decided to completely remove the background averaging. 

I initially thought I was going to just add a warning, then I changed my mind and decided to remove background averaging. Sorry I didn't do a very good job cleaning up after myself: the warning is still in the commit history and in the name of this branch.

ziw-liu commented:
> Sorry I didn't do a very good job cleaning up after myself: the warning is still in the commit history and in the name of this branch.

I wasn't making it clear, but I was suggesting adding the warning back (with different articulation) so that any user who was relying on that bug/feature would be advised of the change we are making. Although the previous behavior was more like a bug, this 'breaking' change still alters the output value of the pipeline if, for example, someone would like to revisit their previous analysis.

talonchandler commented:
Aha! Excellent point. I didn't maintain backwards compatibility. 

I've reverted the changes, added a warning, fixed the bug, and added reminders for what needs to be removed for 1.0.0. 

Thanks Ziwen...great catch.

ziw-liu commented on recOrder/io/utils.py:
```diff
@@ -72,9 +72,12 @@ def load_bg(bg_path, height, width, ROI=None):
+            Warning: earlier versions of recOrder would have averaged over the background ROI. This behavior is 
+            now considered a bug, and future versions of recOrder will not average over the background. 
+            """
+            print(warning_msg)
```
Same question as in [#157](https://github.com/mehta-lab/recOrder/pull/157#discussion_r968971351), as this message may not be visible from the napari window.

talonchandler replied:
Again...good catch. Changed from `print` to `logging.warning`.
